runbotics/runbotics-scheduler/src/scheduler-database/tenant/tenant.controller.ts

```diff
@@ -19,8 +19,12 @@ import { TenantService } from './tenant.service';
 import { TenantInviteCodeDto, tenantInviteCodeSchema } from './dto/invite-code.dto';
 import { CreateTenantDto, createTenantSchema } from './dto/create-tenant.dto';
 import { UpdateTenantDto, updateTenantSchema } from './dto/update-tenant.dto';
+import { Pageable, Paging } from '#/utils/page/pageable.decorator';
+import { Specifiable, Specs } from '#/utils/specification/specifiable.decorator';
+import { TenantCriteria } from './criteria/tenant.criteria';
+import { Tenant as TenantEntity } from './tenant.entity';
 
-@Controller('/api/scheduler/tenants')
+@Controller('/api/scheduler')
```

Comment:
are you sure we want to remove /tenants? It is not being added to the below invite-code e.g. so we have /api/scheduler/invite-code endpoint which for me doesn't make much sense